src/map/script.hpp

```diff
@@ -2090,6 +2090,11 @@ enum e_convertpcinfo_type : uint8 {
 	CPC_ACCOUNT   = 2
 };
 
+enum e_instance_warpall_flag : uint8 {
+	IWA_NOTDEAD   = 0x01,
```

Comment:
```suggestion
	IWA_DEAD   = 0x01,
```
Thats not how bitmasking works... :-(